src/views/UserProfile.vue

```diff
@@ -80,13 +113,37 @@ export default {
       this.processingForm = true
       const formData = {
         userId: this.user.id,
-        name: this.name,
+        name: this.user.name,
       }
       this.user = await this.patchUser(formData)
       this.name = this.user.name
       this.processingForm = false
       // show success?
     },
+    async submitPhoto() {
+      this.processingForm = true
+      const formData = {
+        userId: this.user.id,
+        photoUrl: this.user.photoUrl,
```

Comment:
See other comment, I guess we should submit the `publicId` instead. Maybe you can store it directly in your ActiveStorage setup? Like `user.photo.key = ...`, but that seems a bit unlikely... There might be some other syntax somewhere. Otherwise, just storing the key as a simple string would be enough I think.

Reply:
ah saw these comments after doing the API stuff already. These were pretty linked. But honestly, i think just storing the key as a string on the user worked out just fine 👌